src/libs/actions/Task.js

```diff
@@ -0,0 +1,102 @@
+import CONST from '../../CONST';
+import ONYXKEYS from '../../ONYXKEYS';
+import * as API from '../API';
+import * as ReportUtils from '../ReportUtils';
+import * as Report from './Report';
+
+/**
+ * Assign a task to a user
+ * @param {String} parentReportActionID
+ * @param {String} parentReportID
+ * @param {String} taskReportID
+ * @param {String} name
+ * @param {String} description
+ * @param {String} assignee
+ * @param {String} ownerEmail
+ *
+ */
+
+function assignTask(parentReportActionID, parentReportID, taskReportID, name, description, assignee, assigneeChatReportID, ownerEmail) {
```

Comment:
To keep it consistent, let's call this createTask (and just have the copy reference Assign Task).  You can create a task without assigning it to someone - this kind of makes it seem like you need to assign a task to someone